src/components/Dialogs/Conditions/NumericCondition.js

```diff
@@ -0,0 +1,87 @@
+import i18n from '@dhis2/d2-i18n'
+import { SingleSelectField, SingleSelectOption, Button, Input } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import classes from './styles/Condition.module.css'
+
+const NULL_VALUE = 'NV'
+export const OPERATOR_EQUAL = 'EQ'
+export const OPERATOR_GREATER = 'GT'
+export const OPERATOR_GREATER_OR_EQUAL = 'GE'
+export const OPERATOR_LESS = 'LT'
+export const OPERATOR_LESS_OR_EQUAL = 'LE'
+export const OPERATOR_NOT_EQUAL = '!EQ'
+export const OPERATOR_EMPTY = `EQ:${NULL_VALUE}`
+export const OPERATOR_NOT_EMPTY = `NE:${NULL_VALUE}`
+
+const operators = {
+    [OPERATOR_EQUAL]: 'equal to (=)',
+    [OPERATOR_GREATER]: 'greater than (>)',
+    [OPERATOR_GREATER_OR_EQUAL]: 'greater than or equal to (≥)',
+    [OPERATOR_LESS]: 'less than (<)',
+    [OPERATOR_LESS_OR_EQUAL]: 'less than or equal to (≤)',
+    [OPERATOR_NOT_EQUAL]: 'not equal to (≠)',
+    [OPERATOR_EMPTY]: 'is empty / null',
+    [OPERATOR_NOT_EMPTY]: 'is not empty / not null',
+}
+
+const NumericCondition = ({ condition, onChange, onRemove }) => {
+    const setOperator = input => {
+        if (input.includes(NULL_VALUE)) {
+            onChange(`${input}`)
+        } else {
+            onChange(`${input}:${value || ''}`)
+        }
+    }
+    const setValue = input => onChange(`${operator}:${input || ''}`)
```

Comment:
Same as previous comment. Where is `operator` coming from?

Reply:
See comment above ⏫

Reply:
Same as above